src/frontend/src/components/Main/Feed/Post/Post.js

```diff
@@ -14,7 +14,8 @@ import SharedButton from '../../../../shared/Button/SharedButton'
 import ThreeDots from '../../../../shared/ThreeDots/TreeDots'
 import SmallDot from '../../../../shared/SmallDot/SmallDot'
 import { useDispatch, useSelector } from 'react-redux'
-import { createNewCommentAction, getCommentsForPostAction, toggleLikeAction, getUsersWhoLikedPostAction } from '../../../../redux/Post/postActions'
+import { createNewCommentAction, getCommentsForPostAction, toggleLikeAction } from '../../../../redux/Post/postActions'
+import { getUsersWhoLikedPostAction } from '../../../../redux/User/userActions'
```

Comment:
function getUsersWhoLikedPostAction  should be imported from from '../../../../redux/Post/postActions'